Frontend/Calculator.md

```diff
@@ -0,0 +1,34 @@
+# Interview Technical Assignment - Calculator Interface
+
+We ask all engineers to complete a technical test as part of our recruitment process at NICE. It's a great way for us to assess your technical abilities.
+
+We'll anonymously review your solution before inviting you for interview. Your test would then also form part of the interview itself: we'll discuss your approach, code and tests.
+
+We look for a number of things when we're reviewing, including:
+  - specific front-end coding skills
+  - understanding of things like progressive enhancement, performance and accessibility
+  - use of tooling and the npm ecosystem
+  - how you structure an application.
+
+## Your submission
+
+The details of the [the test](#the-test) are below but *please*:
+
+- provide your solution in **HTML, CSS and JavaScript**
+  - use React if you prefer
+- don't spend more than 3 or 4 hours
+- write appropriate tests
+- use npm packages where needed.
+
+You won't have time to complete everything, but **that's OK**. We're more interested in your approach, thought process, application structure and where you stop, than a complete solution. For example what you prioritise or where you write tests and where you don't.
+
+Submit your solutions via email as a zip file. Please remove any *node_modules* folders and add *.test* to the file extensions so that it can get past email server filters (for example *calculator.zip.test*).
```

Comment:
```suggestion
Submit your solutions via email as a zip file. Please remove any *node_modules* folders and add *.test* to the file extension so that it can get past email server filters (for example *calculator.zip.test*).
```